python/cudf/cudf/core/column/datetime.py

```diff
@@ -316,6 +336,33 @@ def find_last_value(self, value: ScalarLike, closest: bool = False) -> int:
     def is_unique(self) -> bool:
         return self.as_numerical.is_unique
 
+    def isin(self, values: Sequence) -> ColumnBase:
+        if cudf.utils.dtypes.is_scalar(values):
+            raise TypeError(
```

Comment:
Since we have this error in a few different places we might want to factor it out, I think NumPy keeps its common errors in one place and then imports them/raises as needed.

Reply:
Yeah while actually trying to refactor this out, I realized we are double-validating once in `Series` & `DataFrame` and again in `Column`. So removed this check at the `Column` level altogether.